### DISCUSSION TITLE
Form.Item 中的触发规则，无法触发 onBlur

### DISCUSSION THREAD

**Discussion by GRlink** [ant-design/ant-design-mobile#6001] [Q&A]
https://codesandbox.io/s/awesome-joana-pp4pli?file=/app.tsx
![image](https://user-images.githubusercontent.com/30586567/221802378-c784720f-beec-4fd9-9683-d03dca3621da.png)

**Comment by Jackie-llv** [comment]
<img width="1004" alt="image" src="https://github.com/ant-design/ant-design-mobile/assets/28420052/a8d04079-d3fd-40e9-936a-89874a17afc6">
我之前也遇到了这个问题，看了看文档没有'onBlur'这个选项
